File: c/02-condicionais/main032.c
/*Crie um jogo onde o computador vai sortear um número entre 1 e 5 o
jogador vai tentar descobrir qual foi o valor sorteado.

Data 07/10/2025 -> 15ª Dia
*/

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {

    int computador;
    int jogador;

    // Inicializar o gerador de núnero aleatórios

    srand(time(NULL));

    printf("OLA SOU O SEU COMPUTADOR VOU GERAR UM NUMERO ALEATORIO E TENTA ADIVINHAR!\n");

    // Gera número aleatório entre 1 e 5

    computador = rand() % 5 + 1;

    printf("ESCOLHA UM NUMERO ENTRE 1 E 5: ");
    scanf("%d",&jogador);

    if(jogador == computador) {

        printf("VOCE ACERTOU O NUMERO QUE EU GEREI PARABENS!");
    }
    else {

        printf("VOCE PERDEU! O NUMERO QUE EU PENSEI E %d",computador);
    }
    return 0;
}

File: python/02-condicionais/main032.py
'''Crie um jogo onde o computador vai sortear um número entre 1 e 5 o
jogador vai tentar descobrir qual foi o valor sorteado.

Data 07/10/2025 -> 15ª Dia
'''

import random

print("OLÁ SOU O SEU COMPUTADOR VOU GERAR UM NÚMERO ALEATÓRIO E TENTA ADIVINHAR!")
computador = random.randint(1,5)

jogador = int(input('ESCOLHA O NÚMERO: '))

if(jogador == computador):

    print(f'VOCÊ ACERTOU O NÚMERO QUE EU GEREI PARABÉNS!')

else:

    print(f'VOCÊ PERDEU! O NÚMERO QUE EU PENSEI É {computador}')